Dockerfile

```diff
@@ -418,6 +414,14 @@ RUN set -x; \
 	&& cd PageExchange \
     && git checkout -b $MW_VERSION 339056ffba8db1a98ff166aa11f639e5bc1ac665
 
+# This is a commit from master branch
+RUN set -x; \
```

Comment:
Please make this part of the original RUN sentence https://github.com/WikiTeq/docker-wikiteq-mediawiki/pull/66/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L109

Reply:
I can't get how I can separate extensions which should be cloned from REL1_x branches or from master.
As far as I remember we created the script to keep extensions updated, but when I run it I get a hash of commit from REL1_x branch for VEForAll and EditUser, and I see that the hash was changed. How can I see was an extension updated or we should ignore it because we have to clone it from master branch.
it looks like a mess for me

Reply:
That's fine, you were confused by the wrong assumption that the script's purpose is to upgrade extensions when it's not. The script is **not** capable to do it, the script is **not** intended to be used for any sort of upgrades and the only purpose of the script was to generate the initial RUN statement

We will modify the script to make it capable of doing upgrades sometime later when we will have a chance

For this reason, it is ok to modify the statement and clone extensions from `master` like we're doing it for VeForAll [1]. 

1. https://github.com/WikiTeq/docker-wikiteq-mediawiki/blob/master/Dockerfile#L380